actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-es-utils/src/main/resources/mappings/user-mapping.json

```diff
@@ -1,3588 +1,782 @@
-                    "fielddata": true
-                },
-                "updatedDate": {
-                    "type": "text",
-                    "fields": {
-                        "raw": {
-                            "type": "text",
-                            "analyzer": "keylower",
-                            "fielddata": true
-                        }
-                    },
-                    "copy_to": [
-                        "all_fields"
-                    ],
-                    "analyzer": "cs_index_analyzer",
-                    "search_analyzer": "cs_search_analyzer",
-                    "fielddata": true
-                },
-                "userId": {
-                    "type": "text",
-                    "fields": {
-                        "raw": {
-                            "type": "text",
-                            "analyzer": "keylower",
-                            "fielddata": true
-                        }
-                    },
-                    "copy_to": [
-                        "all_fields"
-                    ],
-                    "analyzer": "cs_index_analyzer",
-                    "search_analyzer": "cs_search_analyzer",
-                    "fielddata": true
-                },
-                "yearOfPassing": {
-                    "type": "long",
-                    "fields": {
-                        "raw": {
-                            "type": "long"
-                        }
-                    }
-                }
+          },
+          "analyzer": "cs_search_analyzer",
+          "search_analyzer": "cs_search_analyzer"
+        },
+        "approvalDate": {
+          "type": "text",
+          "fields": {
+            "raw": {
+              "type": "keyword"
             }
-        },
-        "email": {
-            "type": "text",
-            "fields": {
-                "raw": {
-                    "type": "text",
-                    "analyzer": "keylower",
-                    "fielddata": true
-                }
-            },
-            "copy_to": [
-                "all_fields"
-            ],
-            "analyzer": "cs_index_analyzer",
-            "search_analyzer": "cs_search_analyzer",
-            "fielddata": true
-        },
-        "prevUsedEmail": {
-            "type": "text",
-            "fields": {
-                "raw": {
-                    "type": "text",
-                    "analyzer": "keylower",
-                    "fielddata": true
-                }
-            },
-            "copy_to": [
-                "all_fields"
-            ],
-            "analyzer": "cs_index_analyzer",
-            "search_analyzer": "cs_search_analyzer",
-            "fielddata": true
-        },
-        "emailVerified": {
-            "type": "boolean",
-            "fields": {
-                "raw": {
-                    "type": "boolean"
-                }
+          },
+          "analyzer": "cs_search_analyzer",
```

Comment:
Do we need search analyzer for this date field? Also applies to next field.

Reply:
No, We dont need. Actually we can change the type of these field to keyword.